Docs/Sample.plist

```diff
@@ -1204,8 +1204,6 @@
 				<integer>0</integer>
 				<key>SystemAudioVolume</key>
 				<data>Rg==</data>
-				<key>SystemAudioVolumeDB</key>
-				<data>4g==</data>
```

Comment:
I've removed these again, because on machine running old macOS this NVRAM value would be treated as authoritative by OpenCore, but would never be updated by macOS.

Reply:
Yes, makes sense.